src/Logger/WatchdogLogger.php

```diff
@@ -18,7 +19,22 @@
 class WatchdogLogger extends LoggerBase implements LoggerInterface {
 
   /**
+   * The Drupal7 service.
+   *
+   * @var \Drupal\service_container\Legacy\Drupal7
+   */
+  protected $drupal7;
+
+  /**
+   * {@inheritdoc}
```

Comment:
I mean you need to add the doxygen:

+   * @param \Drupal\service_container\Legacy\Drupal7 $drupal7
+   *   The Drupal7 service.